README.md

```diff
@@ -1,35 +1,41 @@
-Description
-===========
+# Description
 
-This cookbook provides the fileutils resource.  The resource can be used to set attributes 
-on all of the files in a directory and subdirectory. We've had years of people 
-complaining that the directory recursive mode, which is in fact a parent operation, does 
-not work on subdirectories and files.  In this cookbook recursive refers to subdirectories
-and the files contained inside them.
+This cookbook provides the fileutils resource.  The resource can be used to set attributes
+on all of the files in a directory and subdirectory. We've had years of people
+complaining that the directory recursive mode, which is in fact a parent operation,
+does not work on subdirectories and files.  In this cookbook recursive refers
+to subdirectories and the files contained inside them.
 
-Requirements
-============
+## Requirements
 
 Developed for RHEL and Solaris servers.
 
+## Resource Parameters
 
-Resource Parameters
-===================
-The fileutils resource will accept two actions.  :change is used to modify ownership and permission mode bit settings.
-:delete is used to remove files and directories. :delete always functions in a recursive mode.  
+The fileutils resource will accept two actions.
+
+* :change is used to modify ownership and permission mode bit settings.
+* :delete is used to remove files and directories.
+  :delete always functions in a recursive mode.
 
 | Action | Parameter | Use
 | ------ | --------- | ---
-| :change | path     | Specify the starting path or file. The path must exist for anything to be done.
+| :change | path     | Specify the starting path or file.
+ The path must exist for anything to be done.
```

Comment:
Wrapping in tables breaks the formatting. Can you please ensure this renders correctly?

Reply:
Renders find for me. What's the issue?

Reply:
It doesn't appear to on GH: https://github.com/sous-chefs/fileutils/tree/ci_testing#resource-parameters

Reply:
Should be good now. Thank you very much for spotting that.